ipn/ipnlocal/local.go

```diff
@@ -4146,6 +4160,8 @@ func (b *LocalBackend) ResetForClientDisconnect() {
 
 func (b *LocalBackend) ShouldRunSSH() bool { return b.sshAtomicBool.Load() && envknob.CanSSHD() }
 
+func (b *LocalBackend) ShouldRunWebClient() bool { return b.webclientAtomicBool.Load() }
```

Comment:
Also check envknob? similar to ssh

Reply:
you would have already had the knob set in order to enable the web client in the first place, but there's probably no harm in checking here also.  We'll be removing the knob before too long anyway.